src/validation.cpp

```diff
@@ -3657,7 +3657,7 @@ bool Chainstate::InvalidateBlock(BlockValidationState& state, CBlockIndex* pinde
     assert(pindex);
     if (pindex->nHeight == 0) return false;
 
-    CBlockIndex* to_mark_failed = pindex;
+    CBlockIndex* invalid_walk_tip = pindex;
```

Comment:
commit message ca2421d52792b32754f1801492822b999839ec3a:
"since the previous commit removes BLOCK_FAILED_CHILD" - only the following commit removes it. Or do you mean "any special logic for BLOCK_FAILED_CHILD"?